MekHQ/src/mekhq/gui/dialog/ContractMarketDialog.java

```diff
@@ -104,16 +103,15 @@ public ContractMarketDialog(Frame frame, Campaign c) {
      */
     private void countSuccessfulContracts() {
         HashMap<String, Integer> successfulContracts = new HashMap<>();
-        for (Mission m : campaign.getMissions()) {
-            if (m.isActive() || !(m instanceof AtBContract)
-                    || ((AtBContract) m).getEmployerCode().equals(campaign.getRetainerEmployerCode())) {
+        for (AtBContract contract : campaign.getAtBContracts()) {
+            if (contract.getStatus().isActive()
```

Comment:
Looks like you missed a .isActiveOn() call here

Reply:
That's a bug